gammapy/spectrum/tests/test_flux_point.py

```diff
@@ -214,10 +213,10 @@ def test_values(self):
         assert_allclose(actual, 6.3298e-11, rtol=1e-2)
 
         actual = flux_points.table["dnde_errn"][0]
-        assert_allclose(actual, 5.9315e-12, rtol=1e-2)
+        assert_allclose(actual, np.nan, rtol=1e-2)
```

Comment:
Do you understand why you get "np.nan" in this test now? Then OK, no need to explain.

Reply:
No, unfortunately I don't understand why this changed. Only the upper limit computation was affected, which seems to be buggy anyway (see #1635). The problem is that the `FluxPointEstimator` is probably the worst piece of code ever written in Gammapy (50% by me...:-) and should be reworked and simplified as soon as possible.  Currently the UL computation uses a custom method with `brentq` and we should just reuse `Minuit`...

Reply:
Then I'd suggest to leave a `TODO` comment line with a link to the commit or PR where this changed (or now to this comment directly), to help the person that will work on the FluxPointEstimator next see what the values previously were.